eppo_core/src/precomputed.rs

```diff
@@ -99,23 +130,42 @@ impl PrecomputedConfiguration {
 
 impl From<PrecomputedConfiguration> for ObfuscatedPrecomputedConfiguration {
     fn from(config: PrecomputedConfiguration) -> Self {
-        let salt = rand::thread_rng().gen();
+        let salt: Str = {
+            let bytes = rand::thread_rng().gen::<[u8; 16]>();
+            base64::prelude::BASE64_STANDARD_NO_PAD
+                .encode(&bytes)
+                .into()
+        };
```

Comment:
This change is to make it use salt in text mode instead of binary as before

Reply:
cc @typotter